include/nil/crypto3/math/polynomial/polynomial_dfs.hpp

```diff
@@ -411,6 +413,31 @@ namespace nil {
                     return result;
                 }
 
+                /**
+                 * Computes the standard polynomial addition, polynomial A += polynomial B, and stores result in
+                 * polynomial C.
```

Comment:
It's slightly more readable if you wrap like this
```suggestion
                 * Computes the standard polynomial addition, polynomial A += polynomial B,
                 * and stores result in polynomial C.
```